appcelerator/titanium_mobile
# fix(android): APK signing will fail when using MD5 keystore and JDK 8+

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26895

---
**MD5 Keystore Test:**
1. Follow the test procedure documented in [TIMOB-26895](https://jira.appcelerator.org/browse/TIMOB-26895).
2. Verify that an APK was successfully built with that ticket's `testmd5.keystore`.

---
**SHA256 Keystore Test:**
1. Copy the `testsha256.keystore` from [TIMOB-26895](https://jira.appcelerator.org/browse/TIMOB-26895) to the root directory of your project.
2. In Appc Studio, select "Package" from the top-left-most dropdown box.
3. In Appc Studio, select "Android Play Store" from the other dropdown box.
4. Click the build button.
5. For "Keystore Location", click the "Browse" button and select the `testsha256.keystore` file.
6. For "Keystore Password", enter: `testsha256`
7. For "Keypair Password", enter: `testsha256`
8. For "Key Alias", enter: `testsha256`
9. Click the "Publish" button.
10. Verify that an APK was successfully built.
